### ISSUE TITLE
Compatibility breaks in 1.17 & 1.17.1 with a location change for NBT data

### ISSUE THREAD

**Issue by brayjamin** [grakkit/server#1]
Error: 

```
[19:53:55] [Server thread/INFO]: [grakkit] Enabling grakkit v4.1.3*
[19:53:57] [Server thread/WARN]: TypeError: Access to host class net.minecraft.server.v1_17_R1.NBTTagByte is not allowed or does not exist.
[19:53:57] [Server thread/WARN]: 	at <js> type(/root/minecraft/servers/cursed-survival/plugins/grakkit/node_modules/@grakkit/core/library/type.js:6:258-272)
[19:53:57] [Server thread/WARN]: 	at <js> type(/root/minecraft/servers/cursed-survival/plugins/grakkit/node_modules/@grakkit/server/library/type.js:8:284-305)
[19:53:57] [Server thread/WARN]: 	at <js> :anonymous(/root/minecraft/servers/cursed-survival/plugins/grakkit/node_modules/@grakkit/server/library/mantle.js:37:2108-2139)
[19:53:57] [Server thread/WARN]: 	at <js> :anonymous(/root/minecraft/servers/cursed-survival/plugins/grakkit/node_modules/@grakkit/server/index.js:30:1398-1424)
[19:53:57] [Server thread/WARN]: 	at <js> :module(plugins/grakkit/index.js:1:25-50)
[19:53:57] [Server thread/WARN]: 	at org.graalvm.polyglot.Context.eval(Context.java:345)
[19:53:57] [Server thread/WARN]: 	at grakkit.Core.open(Core.java:89)
[19:53:57] [Server thread/WARN]: 	at grakkit.Core.init(Core.java:73)
[19:53:57] [Server thread/WARN]: 	at grakkit.Main.onEnable(Main.java:18)
[19:53:57] [Server thread/WARN]: 	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:263)
[19:53:57] [Server thread/WARN]: 	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:370)
[19:53:57] [Server thread/WARN]: 	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:500)
[19:53:57] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_17_R1.CraftServer.enablePlugin(CraftServer.java:518)
[19:53:57] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_17_R1.CraftServer.enablePlugins(CraftServer.java:432)
[19:53:57] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.loadWorld(MinecraftServer.java:712)
[19:53:57] [Server thread/WARN]: 	at net.minecraft.server.dedicated.DedicatedServer.init(DedicatedServer.java:327)
[19:53:57] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.x(MinecraftServer.java:1205)
[19:53:57] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:319)
[19:53:57] [Server thread/WARN]: 	at java.base/java.lang.Thread.run(Thread.java:831)
```

In 1.17, instead of things such as **NBTTagByte** being stored in `net.minecraft.server.<version>.`, they've instead been moved to simply `net.minecraft.nbt.`.
So, instead of @grakkit/server implementing **NBTTagByte** as `net.minecraft.server.v1_17_R1.NBTTagByte`, it should be written as `net.minecraft.nbt.NBTTagByte`.

I would fix this myself and issue a pull request, however I code in JS, not TS lol.

EDIT: The error log shows that this issue also affects @grakkit/core.

**Comment by ghost** [comment]
this issue is unrelated to the `@grakkit/server` or `@grakkit/core` module

**Comment by brayjamin** [comment]
You're very much correct, my apologies. My version of @grakkit/server is outdated. The stacktrace references the module's index file, at line 30. My file contains an import for the mantle module that isn't packaged with @grakkit/server anymore. You discussed this with me earlier and fixed it in a new release of mantle. 

So essentially, I've opened up an issue for something that not only was fixed, shouldn't even be a problem with the @grakkit/server module since the problematic import isn't even packaged inside the module, anymore.
